<?xml version="1.0"?>
<launch>
    <arg name="tested_index" default="0" />
    
    <rosparam>
        key_mappings: ['q','w', 'e', 'r']
        interface_matrix: [[0.9, 0.04, 0.03, 0.03], [0.1, 0.72, 0.06, 0.12], [0.32, 0.02, 0.5, 0.16], [0.005, 0.002, 0.003, 0.99],]
    </rosparam>
    
    <!--rosparam>
        key_mappings: ['q','w']
        interface_matrix: [[0.9, 0.1], [0.2, 0.8]]
    </rosparam-->
    
    <node pkg="lthmi_nav" type="emulator_keyboard.py" 
        name="hmi_emulator_keyboard" >
        <remap from="~key_mappings" to="/key_mappings"/>
        <remap from="~interface_matrix" to="/interface_matrix"/>
    </node>

    
    <node pkg="lthmi_nav" type="emulator_keyboard_test.py" 
        name="hmi_emulator_keyboard_test" >
        <param name="tested_index" value="$(arg tested_index)" />
        <remap from="~interface_matrix" to="/interface_matrix"/>
    </node>
    
</launch>
